<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRoadUser</class>
 <widget class="QDialog" name="EditRoadUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>401</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="lat_label">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Latitude</string>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="lat_spin">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>-36000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>36000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lon_label">
        <property name="text">
         <string>Longitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="lon_spin">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="speed_label">
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="speed_spin">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="head_label">
        <property name="text">
         <string>Heading</string>
        </property>
        <property name="margin">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="head_spin">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedKingdom"/>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditRoadUser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
